lms/djangoapps/instructor/services.py

```diff
@@ -44,15 +45,12 @@ def delete_student_attempt(self, student_identifier, course_id, content_id, requ
         course_id = CourseKey.from_string(course_id)
 
         try:
-            student = get_student_from_identifier(student_identifier)
+            student = get_user_by_username_or_email(student_identifier)
```

Comment:
Nice.

Reply:
Yeah, looked into the `get_student_from_identifier` and it's a deprecated method that just calls this function. So figured worth updating.